invenio/modules/accounts/templates/accounts/settings/helpers_base.html

```diff
@@ -1,6 +1,6 @@
 {#
 # This file is part of Invenio.
-# Copyright (C) 2014 CERN.
+# Copyright (C) 2015 CERN.
```

Comment:
You should never remove past years but only append the current one: `# Copyright (C) 2014, 2015 CERN.`